apps/web/playwright/booking-pages.e2e.ts

```diff
@@ -388,7 +389,7 @@ test.describe("Booking round robin event", () => {
     const schedule: Schedule = [[], [dateRanges], [dateRanges], [dateRanges], [dateRanges], [dateRanges], []];
 
     const testUser = await users.create(
-      { username: "test-user", name: "Test User", email: "testuser@example.com", schedule },
```

Comment:
This was causing an unique constrain conflict when testing both legacy en future.